src/libraries/Microsoft.Extensions.Configuration.UserSecrets/src/UserSecretsConfigurationExtensions.cs

```diff
@@ -169,6 +169,9 @@ public static IConfigurationBuilder AddUserSecrets(this IConfigurationBuilder co
         /// <param name="reloadOnChange">Whether the configuration should be reloaded if the file changes.</param>
         /// <returns>The configuration builder.</returns>
         public static IConfigurationBuilder AddUserSecrets(this IConfigurationBuilder configuration, string userSecretsId, bool reloadOnChange)
+            => AddUserSecretsInternal(configuration, userSecretsId, true, reloadOnChange);
```

Comment:
nit, usually we name bool parameters. It is not a big deal, but for next time. It helps understand what the true is for when reading the call site.